.travis.yml

```diff
@@ -28,7 +28,7 @@ addons:
 
 before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update           ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install qt4      ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then sh mac-build.sh       ; fi
```

Comment:
What is the problem with installing `qt4` using *brew*? I personally prefer the current method.